2018/src/confcal/interactive.confcal

```diff
@@ -0,0 +1,27 @@
+Interactive
+on 2018/11/24
+at Yahoo Japan Corporation#ChIJ30G_l3iLGGARWbrLEBhHS3s
+
+[Room A]
+10:00-10:30 Doors Open
+10:30-11:45 Node/JS Discussion
+13:00-15:00 Code And Learn by hiroppy
+15:00-17:00 JavaScript TodoMVC Workshop by Yosuke Furukawa
+17:00-18:00 Lightning Talks
+    - Serverless in Scala on Node.js: Get Fast and Productive by Tatsuno Yasuhiro
+    - 私の生活を変えたHeadless Chrome by 大島　光貴
+    - Deploy TypeScript with CodePipeline in Lambda by 石川　将吾
```

Comment:
The Speaker wants to change the title as below.
Would you change it?


```
before: Deploy TypeScript with CodePipeline in Lambda
after:    Deploy TypeScript with CodePipeline in Fargate
```

Reply:
Done in https://github.com/nodefest/nodefest.github.com/pull/139#pullrequestreview-166411996